modules/sugar/src/main/ts/ephox/sugar/api/properties/Css.ts

```diff
@@ -155,9 +157,10 @@ const copy = (source: Element<DomNode>, target: Element<HTMLElement>) => {
  * do not rely on this return value.
  * It's here so the closure compiler doesn't optimise the property access away.
  */
-const reflow = (e: Element<HTMLElement>) => e.dom().offsetWidth;
+const reflow = (e: Element<HTMLElement>): number =>
```

Comment:
This should be void, as we don't want anyone to use the output of this.

Reply:
What a strange function. Is the comment valid? Will a compiler / minifier optimize this away?

Reply:
We don't use the `closure` compiler anymore I believe, so the comment may no longer be valid. Given it is something that has a potential side-effect, I'd be surprised if it's intentionally being stripped.

Reply:
I checked using the current build stack (rollup and uglifyjs) and it appears we don't need it to return any more as it is retained:

**tinymce.js**:
```
var reflow = function (e) {
  e.dom().offsetWidth;
};
...
var doc = Element.fromDom(editor.getDoc());
reflow(body);
var marker = createMarker(Element.fromDom(rng.startContainer), rng.startOffset);
```

**tinymce.min.js**:
```
i=Ct.fromDom(e.getDoc());o.dom().offsetWidth;var a=Lh(Ct.fromDom(n.startContainer),n.startOffset);
```

**themes/silver/theme.js**:
```
var reflow = function (e) {
  e.dom().offsetWidth;
};
...
var setShrunk = function (component, slideConfig) {
  remove$4(component.element(), slideConfig.openClass);
  add$2(component.element(), slideConfig.closedClass);
  set$3(component.element(), getDimensionProperty(slideConfig), '0px');
  reflow(component.element());
};
```

**themes/silver/theme.min.js**:
```
qi=function(t){t.dom().offsetWidth}
...
C_=function(t,n){Ci(t.element(),n.openClass),Si(t.element(),n.closedClass),Ni(t.element(),w_(n),"0px"),qi(t.element())}
```

Reply:
In saying that though, we can't guarantee it'll always work and we can't guarantee it'll always be rollup+uglifyjs, so it'd probably be pretty risky to make it actually not return. So it may just be safer to cast it, eg:
```
const reflow = (e: Element<HTMLElement>): void =>
  e.dom().offsetWidth as unknown as void;
```

Reply:
Ok. I've gone with the cast.